src/Microsoft.FeatureManagement/ConfigurationFields.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+namespace Microsoft.FeatureManagement
+{
+    internal static class ConfigurationFields
+    {
+        // Enum keywords
+        public const string RequirementType = "RequirementType";
+
+        // Feature filters keywords
+        public const string FeatureFiltersSectionName = "EnabledFor";
+        public const string FeatureFilterConfigurationParameters = "Parameters";
+
+        // Other keywords
+        public const string NameKeyword = "Name";
+        public const string FeatureManagementSectionName = "FeatureManagement";
+        public const string FeatureFlagsSectionName = "FeatureFlags";
+
+        // Azure App Configuration feature flag schema keywords
+        public const string IdKeyword = "id";
```

Comment:
Id, Enabled, Conditions and ClientFilters are renamed.
 
I want to remain the naming of NameKeyword because it is consistent with the preview branch. https://github.com/microsoft/FeatureManagement-Dotnet/blob/preview/src/Microsoft.FeatureManagement/ConfigurationFields.cs#L38

Reply:
It's for internal use only, why we can't update it?

Reply:
Changed to "Name" and "LowercaseName".